### DISCUSSION TITLE
Winget.exe Version

### DISCUSSION THREAD

**Discussion by Frank-Roedel** [microsoft/winget-cli#2689] [Q&A] [answered]
Is there a way to use or call a different Winget version than the one we have installed on the system? 

For example, the Windows has a newer version Winget installed such as v1.3.2691 but we would like to use Winget v1.2.10271 to install a specific application.

**Comment by denelon** [comment, marked as answer]
No, the App Installer (the package WinGet is distributed in) doesn't support "side-by-side" installation. One would need to manually "downgrade" App Installer to be able to use a previous version, and then upgrade it again to get back to the current desired version.
